design/work_item_children.go

```diff
@@ -0,0 +1,23 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+var _ = a.Resource("work-item-children", func() {
+	a.Parent("workitem")
+
+	a.Action("list", func() {
```

Comment:
Please move the list action to `design/workitems.go` like so, because we don't need another controller just for listing the children.

```go
	a.Action("show", func() {
		a.Routing(
			a.GET("/:id/children"),
		)
		a.Description("List children associated with the given work item.")
		a.Params(func() {
			a.Param("id", d.String, "id")
		})
		a.Response(d.OK, func() {
			a.Media(workItemList)
		})
		a.Response(d.BadRequest, JSONAPIErrors)
		a.Response(d.InternalServerError, JSONAPIErrors)
		a.Response(d.NotFound, JSONAPIErrors)
	})
```